oak_abi/src/label/mod.rs

```diff
@@ -66,8 +66,18 @@ impl crate::proto::oak::label::Label {
         // The target label must have (compared to the self label):
         // - same or more confidentiality tags
         // - same or fewer integrity tags
-        self_confidentiality_tags.is_subset(&other_confidentiality_tags)
-            && other_integrity_tags.is_subset(&self_integrity_tags)
+        (other_confidentiality_tags.contains(&Tag::universal_top_tag())
```

Comment:
Updated this check for completeness, but I am not sure if we ever want to use the top tag as part of a label (perhaps when we switch to robust declassification and transparent endorsement?). Currently, I am only using it to set the privilege of HTTP and gRPC server nodes.

Reply:
LGTM, @aferr should take a look too, please update the comment as well.

Reply:
Done.